extensions/kubernetes/vanilla/deployment/src/main/java/io/quarkus/kubernetes/deployment/KubernetesConfig.java

```diff
@@ -340,11 +340,20 @@ public enum DeploymentResourceKind {
      * Flag to enable init task externalization.
      * When enabled (default), all initialization tasks
      * created by extensions, will be externalized as Jobs.
-     * In addition the deployment will wait for these jobs.
+     * In addition, the deployment will wait for these jobs.
      */
     @ConfigItem(defaultValue = "true")
     boolean externalizeInit;
 
+    /**
+     * List of init tasks to be disabled and hence not to be generated. The init tasks are automatically generated by
+     * extensions like Flyway to perform the database migration before staring up the application.
+     *
+     * This property is only taken into account if `quarkus.kubernetes.externalize-init` is true.
+     */
+    @ConfigItem
+    Optional<List<String>> disableInitTasks;
```

Comment:
I would suggest creating a new config group called: `InitTaskConfig` or something and move there:
- `disableInitTasks` -> `disabledTasks`
- `externalizeInit` -> `generate` or `enabled` or something like this. The original one should be deprecated.

Why? 
Because I can see use in the future adding other stuff there too, like the default image, timeouts and more. So, we better address that now.

This doesn't necesserily need to be part of this pull request. It's more like a suggestion.

Reply:
I think this makes total sense. Many thanks for this suggestion! Will change it as part of this pull request.

Reply:
PR updated with your suggestion, let me know if this was what you had in mind. 
Btw, I took this opportunity to allow configuring the init container image.